pkg/docker/docker.go

```diff
@@ -87,7 +98,7 @@ func (step Step) Exec() (*io.ReadCloser, error) {
 		ctx,
 		&container.Config{
 			Image:      step.Image,
-			Cmd:        step.Command,
+			Cmd:        []string{"tail", "-f", "/dev/null"},
```

Comment:
Can you explain why we are passing `tail` as the command? Can we safely assume all the dockers have `tail`?

Reply:
The fundamental base is `scratch`, which doesn't even have the `ls` command. After that, the most minimalistic base images are `busybox` and `alpine`, and both of them contain the `tail` command.